### PR TITLE
[MIG] web_widget_timepicker: Migration to 12.0

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: web_widget_timepicker/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# Copyright 2016 Vividlab (<http://www.vividlab.de>)
```
Comment (BT-dherreros) [inline web_widget_timepicker/__manifest__.py]: add braintec as copyright 2020

File: web_widget_timepicker/__manifest__.py [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+{
+    "name": "Web Timepicker Widget",
+    "version": "12.0.1.0.0",
+    "author": "VividLab, "
```
Comment (BT-dherreros) [inline web_widget_timepicker/__manifest__.py:6]: add us as authors too

File: web_widget_timepicker/readme/CONTRIBUTORS.rst [hunk 3]
```diff
@@ -0,0 +1,3 @@
+* Michael Fried <Michael.Fried@vividlab.de>
+* Kaushal Prajapati <kbprajapati@live.com>
+* Alexandre Díaz <dev@redneboa.es>
```
Comment (BT-dherreros) [inline web_widget_timepicker/readme/CONTRIBUTORS.rst:3]: Add yourself as a contributor

File: web_widget_timepicker/static/description/index.html [hunk 4]
```diff
@@ -0,0 +1,456 @@
+<?xml version="1.0" encoding="utf-8" ?>
```
Comment (BT-dherreros) [inline web_widget_timepicker/static/description/index.html]: this line can be removed in v12

File: web_widget_timepicker/static/description/index.html [hunk 5]
```diff
@@ -0,0 +1,456 @@
...
+!! This file is generated by oca-gen-addon-readme !!
+!! changes will be overwritten.                   !!
+!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!! -->
+<p><a class="reference external" href="https://odoo-community.org/page/development-status"><img alt="Beta" src="https://img.shields.io/badge/maturity-Beta-yellow.png" /></a> <a class="reference external" href="http://www.gnu.org/licenses/agpl-3.0-standalone.html"><img alt="License: AGPL-3" src="https://img.shields.io/badge/licence-AGPL--3-blue.png" /></a> <a class="reference external" href="https://github.com/OCA/web/tree/11.0/web_widget_timepicker"><img alt="OCA/web" src="https://img.shields.io/badge/github-OCA%2Fweb-lightgray.png?logo=github" /></a> <a class="reference external" href="https://translation.odoo-community.org/projects/web-11-0/web-11-0-web_widget_timepicker"><img alt="Translate me on Weblate" src="https://img.shields.io/badge/weblate-Translate%20me-F47D42.png" /></a> <a class="reference external" href="https://runbot.odoo-community.org/runbot/162/11.0"><img alt="Try me on Runbot" src="https://img.shields.io/badge/runbot-Try%20me-875A7B.png" /></a></p>
```
Comment (BT-dherreros) [inline web_widget_timepicker/static/description/index.html:369]: this should be regenerated, I have no clue on how to do it though. Paging @BT-rmartin for more information.

File: web_widget_timepicker/static/src/js/web_widget_timepicker.js [hunk 6]
```diff
@@ -0,0 +1,96 @@
+/* Copyright 2016 Vividlab <http://www.vividlab.de>
+ * Copyright 2017 Kaushal Prajapati <kbprajapati@live.com>
+ * Copyright 2019 Alexandre Díaz <dev@redneboa.es>
```
Comment (BT-dherreros) [inline web_widget_timepicker/static/src/js/web_widget_timepicker.js:3]: and braintec here as a contributor

File: web_widget_timepicker/views/web_widget_timepicker_assets.xml [hunk 7]
```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="utf-8"?>
```
Comment (BT-dherreros) [inline web_widget_timepicker/views/web_widget_timepicker_assets.xml]: This line can be removed in v12

**Review by BT-dherreros — CHANGES_REQUESTED** [review]
Remember to test the widget against its correct behavior

**Comment by ghost** [comment]
added corrections

File: web_widget_timepicker/static/description/index.html (hunk 5 above)
Comment (BT-dherreros) [reply web_widget_timepicker/static/description/index.html:369]: Check this, ask Raul about it.

File: web_widget_timepicker/static/src/js/web_widget_timepicker.js (hunk 6 above)
Comment (BT-dherreros) [reply web_widget_timepicker/static/src/js/web_widget_timepicker.js:3]: You forgot to add yourself as contributor

**Review by BT-dherreros — CHANGES_REQUESTED** [review]

**Pull request closed without merging**